### ISSUE TITLE
 Just want to share some time info

### ISSUE THREAD

**Issue by MartinGao** [madMAx43v3r/chia-plotter#110]
First thanks for your amazing job!!!

This is not an actual issue but just some time info that might be helpful to the community

My setup is dual Xeon(R) E5-2650@2.00GHz R620 with 64GB RAM and 2.5-inch SAS 900GB 15K HDD x 8 (no RAID)
I tried to run 8 jobs in parallel without RAM drive as tmp2. All 8 jobs finished at the same time and each job took `7.5 hrs`.
As for comparison, the official plotter with the same setup and config takes about `12hrs` for one job.

`12hrs` -> `7.5hrs` that's about 40% faster!

And more importantly, no need expensive/potentially vulnerable SSD or a bunch of RAM, the plot time improvement with HDD is also amazing.

**Comment by carlfarrington** [comment]
Hi. Are you saying you got all 8 jobs done in 7.5 hours? so under 1hr per plot? with HDD?
It's 1 HDD per plot I guess to stop the drives thrashing themselves like crazy?

**Comment by VertiHydro** [comment]
> Hi. Are you saying you got all 8 jobs done in 7.5 hours? so under 1hr per plot? with HDD?
> It's 1 HDD per plot I guess to stop the drives thrashing themselves like crazy?

No - he's saying that each job took 7.5 hours, but he was running 8 jobs at once, in parallel.

**Comment by carlfarrington** [comment]
> > Hi. Are you saying you got all 8 jobs done in 7.5 hours? so under 1hr per plot? with HDD?
> > It's 1 HDD per plot I guess to stop the drives thrashing themselves like crazy?
> 
> No - he's saying that each job took 7.5 hours, but he was running 8 jobs at once, in parallel.

Yeah. So 8 jobs done in 7.5hrs. Average of 57 minutes per plot?
I suppose with 1 drive per job and many cores available, it makes sense.
Still good that you could plot to HDD in 7.5 hrs. It takes 6hr30 on NVMe SSD with std plotter for me, and that's with much better single core performance (i5-9500).

**Comment by VertiHydro** [comment]
> > > Hi. Are you saying you got all 8 jobs done in 7.5 hours? so under 1hr per plot? with HDD?
> > > It's 1 HDD per plot I guess to stop the drives thrashing themselves like crazy?
> > 
> > 
> > No - he's saying that each job took 7.5 hours, but he was running 8 jobs at once, in parallel.
> 
> Yeah. So 8 jobs done in 7.5hrs. Average of 57 minutes per plot?
> I suppose with 1 drive per job and many cores available, it makes sense.
> Still good that you could plot to HDD in 7.5 hrs. It takes 6hr30 on NVMe SSD with std plotter for me, and that's with much better single core performance (i5-9500).

Yep, average of 57 mins per plot. And yeah, exactly, 6-7 hours is about the average time for the official plotter on an NVMe SSD, and now we can do it with HDDs! Insane!

**Comment by carlfarrington** [comment]
I can plot on my farmer now I think. Usually I plot elsewhere and FTP plots across. lovely. The farmer has lots of HDDs :)

**Comment by MartinGao** [comment]
Yep
`running 8 jobs at once in parallel, each job took 7.5 hours, average of 57 mins per plot`

I did this only because the SSD price was way too high and I really did not want to `waste` my money to purchase SSDs only for plotting. 

Both the R620 and plotting HDDs are used. HDD is about $15 for 300GB 15K SAS. R620 is about $400.
400 + 15 * 8 = 520 USD.

520 USD => 16 plots per day (official plotter), it seems reasonable to me.

**Comment by vazmuten** [comment]
If plotting of the farmer on the HDDs this could slow the reponce time ...

**Comment by carlfarrington** [comment]
> If plotting on the farmer PC on the HDDs => this could slow the farmer reponce time ...

Yep it is something I will watch for. Thanks 👍

**Comment by Rickyose** [comment]
> 
> 
> First thanks for your amazing job!!!
> 
> This is not an actual issue but just some time info that might be helpful to the community
> 
> My setup is dual Xeon(R) [E5-2650@2.00GHz](mailto:E5-2650@2.00GHz) R620 with 64GB RAM and 2.5-inch SAS 900GB 15K HDD x 8 (no RAID)
> I tried to run 8 jobs in parallel without RAM drive as tmp2. `./chia_plot -n 1 -r 3 -u 7 ...`
> All 8 jobs finished at the same time and each job took `7.5 hrs`.
> 
> As for comparison, the official plotter with the same setup and config takes about `12hrs` for one job.
> 
> `12hrs` -> `7.5hrs` that's about 40% faster!
> 
> And more importantly, no need expensive/potentially vulnerable SSD or a bunch of RAM, the plot time improvement with HDD is also amazing.

Thats very fast, can you tell us what your complete command, because i am a bit confused

./chia_plot -n 1 -r 3 -u 7

Is that all command  do you need, to replicate yours? how do you set temp drive?
Thank you

**Comment by MartinGao** [comment]
@Rickyose 

the complete command should be like `./chia_plot -n 1 -r 3 -u 7 -t /path/to/your/tmp/hdd/ -d /path/to/your/final/hdd/ -p your_pool_public_key -f your_farmer_public_key`
